src/render/glyph_manager.js

```diff
@@ -218,15 +212,20 @@ class GlyphManager {
             } else if (/light/i.test(stack)) {
                 fontWeight = '200';
             }
-            tinySDF = entry.tinySDF = new GlyphManager.TinySDF(24 * SDF_SCALE, 3 * SDF_SCALE, 8 * SDF_SCALE, .25, family, fontWeight);
+
+            const fontSize = 24 * SDF_SCALE;
+            const buffer = 3 * SDF_SCALE;
+            const radius = 8 * SDF_SCALE;
+            tinySDF = entry.tinySDF = new GlyphManager.TinySDF({fontFamily, fontWeight, fontSize, buffer, radius});
+            tinySDF.fontWeight = fontWeight;
```

Comment:
Just curious, why do we need to set `tinySDF.fontWeight` when `fontWeight` is passed into the constructor?

Reply:
@ryanhamley it was never a part of public `TinySDF` API, it just happened to be exposed there but is no longer the case with v2.